src/authenticationinapp/authenticationinapp.cpp

```diff
@@ -16,28 +16,12 @@ constexpr int PASSWORD_MIN_LENGTH = 8;
 
 namespace {
 Logger logger(LOG_MAIN, "AuthenticationInApp");
-AuthenticationInApp* s_instance = nullptr;
 }  // namespace
 
 // static
-AuthenticationInApp* AuthenticationInApp::instance() {
-  if (!s_instance) {
-    new AuthenticationInApp(qApp);
-  }
-  Q_ASSERT(s_instance);
-  return s_instance;
-}
-
-AuthenticationInApp::AuthenticationInApp(QObject* parent) : QObject(parent) {
-  MVPN_COUNT_CTOR(AuthenticationInApp);
```

Comment:
you can remove 'leakdetector.h' everywhere you remove the MVPN_COUNT_CTOR/DTOR macros.